test/addons/velero/test

```diff
@@ -0,0 +1,162 @@
+#!/usr/bin/env python3
+
+# SPDX-FileCopyrightText: The RamenDR authors
+# SPDX-License-Identifier: Apache-2.0
+
+import json
+import os
+import sys
+import time
+
+from drenv import kubectl
+from drenv import commands
+
+
+BACKUP = "nginx-backup"
+
+
+def test(cluster):
+    print("Deploying nginx example")
+    kubectl.apply("--filename", "nginx.yaml", context=cluster)
+
+    print("Waiting until nginx is rolled out")
+    kubectl.rollout(
+        "status",
+        "deploy/nginx-deployment",
+        "--namespace=nginx-example",
+        "--timeout=180s",
+        context=cluster,
+    )
+
+    print("Deleting older backups")
+    if get_backup(cluster, BACKUP):
+        delete_backup(cluster, BACKUP)
```

Comment:
Maybe we need to check inside the delete_backup, if there are some older backups to delete.